### PR TITLE
TS - People

### PR DESCRIPTION
**Pull request by buoyad**
r? @keybase/react-hackers

### CODE REVIEW

File: shared/tsconfig.json [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
-        "node_modules",
-        "react-native",
-        "desktop/dist"
-    ]
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "sourceMap": true,
+    "noImplicitAny": false,
+    "module": "commonjs",
+    "moduleResolution": "node",
```
Comment (buoyad) [inline shared/tsconfig.json]: Only added this, the rest is formatting. This is to support `import moment from 'moment'` and should match our reality better in general

Comment (MarcoPolo) [reply shared/tsconfig.json]: o cool! I had a different solution that used the `"allowSyntheticDefaultImports": true,` flag. I guess this is better?

Comment (MarcoPolo) [reply shared/tsconfig.json]: O hmm, I think we actually need this as well

Comment (buoyad) [reply shared/tsconfig.json]: Yeah, I saw that one first too

File: shared/people/index.shared.tsx [hunk 2]
```diff
@@ -7,10 +6,13 @@ import Todo from './todo/container'
 import FollowNotification from './follow-notification'
 import Announcement from './announcement/container'
 import FollowSuggestions from './follow-suggestions'
-import {type Props} from '.'
+import {Props} from '.'
 import AirdropBanner from '../wallets/airdrop/banner/container'
 
-export const itemToComponent: (Types.PeopleScreenItem, Props) => React.Node = (item, props) => {
+export const itemToComponent: (arg0: Types.PeopleScreenItem, props: Props) => React.ReactNode = (
```
Comment (adamjspooner) [inline shared/people/index.shared.tsx]: Why `arg0` and not `item`?

Comment (buoyad) [reply shared/people/index.shared.tsx]: That was from the converter, I'll change

**Comment by buoyad** [comment]
@keybase/react-hackers rebased. PTAL

**Pull request merged**